packages/wrangler/src/proxy.ts

```diff
@@ -1,14 +1,15 @@
 import { createServer as createHttpServer } from "node:http";
 import { connect } from "node:http2";
 import { createServer as createHttpsServer } from "node:https";
+import https from "node:https";
 import { networkInterfaces } from "node:os";
-import WebSocket from "faye-websocket";
 import { createHttpTerminator } from "http-terminator";
 import { useEffect, useRef, useState } from "react";
 import serveStatic from "serve-static";
 import { getHttpsOptions } from "./https-options";
 import { logger } from "./logger";
 import type { CfPreviewToken } from "./create-worker-preview";
+import type WebSocket from "faye-websocket";
```

Comment:
dumb question, isn't `WebSocket` just a native TypeScript type?

Reply:
I don't think WebSocket is a native type - although looks like the `WebSocket` type from `faye-websocket` is actually just any... will go and fix the typing to avoid faye-websocket

Reply:
Have how fixed the typing, and removed the dependence on faye-websocket completely